### DISCUSSION TITLE
[Feature] Plausibility check for images of a person with a date of birth and a quick way to remove all images before the day of birth

### DISCUSSION THREAD

**Discussion by kingvuke** [immich-app/immich#5969] [Feature Request]
### The feature

[Feature] Plausibility check for images of a person with a date of birth and a quick way to remove all images before the day of birth.

My daughter was born in 2019. Face recognition has assigned 100s of pictures from before her birth to her.
A quick and easy way to remove all these pictures of the person would be helpful.

### Platform

- [x] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #4743
